changelogs/fragments/remove_deprecated.yaml

```diff
@@ -1,9 +0,0 @@
----
-major_changes:
-  - "This release removes all deprecated plugins that have reached their end-of-life."
-  - "This includes the following modules:"
-  - "ios_interface"
-  - "ios_l2_interface"
-  - "ios_l3_interface"
-  - "ios_static_route"
-  - "ios_vlan"
```

Comment:
This is an unordered list, so I would make each line independent like https://github.com/ansible-collections/ansible.netcommon/pull/458/files#diff-6972579239b23077b784645098c3934bc3e38a58dee82ae456b9d83ab884d3f4